### PR TITLE
fix: Fixing Font variant not rendering in JS customizer issue.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - Trello task - https://trello.com/c/C9T32X9I/219-debug-fix-font-variant-issue-in-js-customizer

### Types of changes
Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?
 - Font variant rendering in JS customizer.
 - Also, check Font family & Font weight controls with it.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/custom-controls/typography/class-astra-control-typography.php [hunk 1]
```diff
@@ -280,8 +157,15 @@ protected function content_template() {
...
+		foreach ( $values as $key => $value ) {
+			echo '<option value="' . esc_attr( $value ) . '" selected="selected" >' . esc_html( $value ) . '</option>';
+		}
+		?>
+		<input class="ast-font-variant-hidden-value" type="hidden" value="{{data.value}}">
 		</select>
-
-		<?php
+		<?php 
+		echo '<span class="ast-control-tooltip dashicons dashicons-editor-help ast-variant-description" title="Only selected Font Variants will be loaded from Google Fonts."></span>';
```
Comment (vrundakansara) [inline inc/customizer/custom-controls/typography/class-astra-control-typography.php:169]: Translate this `title`.

**Comment by premanshup** [comment]
@RajkiranBagal Position of the font-variant is not as expected, please check. It should come after the Font Family. - https://a.cl.ly/YEuo9r8O

File: inc/customizer/custom-controls/typography/class-astra-control-typography.php (hunk 1 above)
Comment (RajkiranBagal) [reply inc/customizer/custom-controls/typography/class-astra-control-typography.php]: Done

**Comment by RajkiranBagal** [comment]
> @RajkiranBagal Position of the font-variant is not as expected, please check. It should come after the Font Family. - https://a.cl.ly/YEuo9r8O

Fixed.

**Review by premanshup — APPROVED** [review]

**Pull request merged**